pages/Documents/RichMessaging/ViberTemplates/quick-replies-template.md

```diff
@@ -0,0 +1,135 @@
+---
+pagename: Quick Replies Template
+redirect_from:
+- quick-replies-viber-template.html
+  Keywords: quick replies rich messaging
+  sitesection: Documents
+  categoryname: "Rich Messaging"
+  documentname: Viber Templates
+  permalink: viber-templates-quick-replies-template.html
+  root-link: true
+  indicator: messaging
+---
+
+### Overview
+
+Quick Replies for Viber include the question/statement text and a set of up to 11 replies that each contain:
```

Comment:
@joshuajott can you mention that quick replies will be presented to the user as custom keyboard?